src/app/shared/table-of-contents/table-of-contents.html

```diff
@@ -1,8 +1,8 @@
-<div *ngIf="links?.length" class="docs-toc-container">
-  <div class="docs-toc-heading">Contents</div>
+<div *ngFor="let linkSection of _linkSections" class="docs-toc-container">
+  <div class="docs-toc-heading">{{linkSection.name}}</div>
   <nav>
     <a [href]="_rootUrl + '#' + link.id"
```

Comment:
Can we rename the harness `id`s to be `testing-classes` and `testing-interfaces`?

Currently, clicking on the links under "Testing" for "Classes" or "Interfaces" scrolls to the Component's Classes and Interfaces, not the Harness' Classes and Interfaces.

Reply:
Absolutely - I have a note to make sure we change the `dgeni` implementation so we do a better job of making the IDs unique in each doc (probably prefixed with the package name)